README.md

```diff
@@ -202,10 +202,12 @@ If you register the service providers in the correct order in your container (A
 
 - [Simplex](https://github.com/mnapoli/simplex): A [Pimple 3](https://github.com/silexphp/Pimple) fork with full [container-interop](https://github.com/container-interop/container-interop) compliance and cross-framework service-provider support.
 - [Service provider bridge bundle](https://github.com/thecodingmachine/service-provider-bridge-bundle): Use container-interop's service-providers into a Symfony container.
+- [Yaco](https://github.com/thecodingmachine/yaco): A compiler that generates container-interop compliant containers. Yaco can consume service-providers.
 
 ### Packages providing *service providers*
 
 - [Glide Module](https://github.com/mnapoli/glide-module): A module integrating Glide in an application using a service provider.
+- [Stratigility Module](https://github.com/thecodingmachine/stratigility-harmony): A service provider for the Stratigility PSR-7 middleware.
```

Comment:
Not related to the PR but FYI since we've discussed the "harmony" name I've discovered this package: https://github.com/woohoolabs/harmony